[BUG] databricks nightly deploy failed

could be related to https://github.com/NVIDIA/spark-rapids/pull/3523

**Describe the bug**

spark301db,
```
[2021-10-16T07:21:25.968Z] [INFO] ------------------------------------------------------------------------
[2021-10-16T07:21:25.968Z] [INFO] Reactor Summary:
[2021-10-16T07:21:25.968Z] [INFO] 
[2021-10-16T07:21:25.968Z] [INFO] RAPIDS Accelerator for Apache Spark Root Project ... FAILURE [  0.802 s]
[2021-10-16T07:21:25.968Z] [INFO] RAPIDS Accelerator for Apache Spark SQL Plugin ..... SKIPPED
[2021-10-16T07:21:25.968Z] [INFO] RAPIDS Accelerator for Apache Spark Shuffle Plugin . SKIPPED
[2021-10-16T07:21:25.968Z] [INFO] RAPIDS Accelerator for Apache Spark SQL Plugin Shims SKIPPED
[2021-10-16T07:21:25.968Z] [INFO] RAPIDS Accelerator for Apache Spark SQL Plugin Spark 3.0.1 Shim SKIPPED
[2021-10-16T07:21:25.968Z] [INFO] RAPIDS Accelerator for Apache Spark Scala UDF Plugin SKIPPED
[2021-10-16T07:21:25.968Z] [INFO] RAPIDS Accelerator for Apache Spark Aggregator ..... SKIPPED
[2021-10-16T07:21:25.968Z] [INFO] RAPIDS Accelerator for Apache Spark Distribution ... SKIPPED
[2021-10-16T07:21:25.968Z] [INFO] RAPIDS Accelerator for Apache Spark UDF Examples ... SKIPPED
[2021-10-16T07:21:25.968Z] [INFO] RAPIDS Accelerator for Apache Spark Tests .......... SKIPPED
[2021-10-16T07:21:25.968Z] [INFO] rapids-4-spark-integration-tests_2.12 .............. SKIPPED
[2021-10-16T07:21:25.968Z] [INFO] rapids-4-spark-api-validation ...................... SKIPPED
[2021-10-16T07:21:25.968Z] [INFO] ------------------------------------------------------------------------
[2021-10-16T07:21:25.968Z] [INFO] BUILD FAILURE
[2021-10-16T07:21:25.968Z] [INFO] ------------------------------------------------------------------------
[2021-10-16T07:21:25.968Z] [INFO] Total time: 1.070 s
[2021-10-16T07:21:25.968Z] [INFO] Finished at: 2021-10-16T07:21:25+00:00
[2021-10-16T07:21:26.223Z] [INFO] Final Memory: 19M/685M
[2021-10-16T07:21:26.223Z] [INFO] ------------------------------------------------------------------------
[2021-10-16T07:21:26.223Z] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.7:deploy-file (default-cli) on project rapids-4-spark-parent: /home/jenkins/agent/workspace/rapids_databricks_nightly-dev-github-301-180/deploy/spark-rapids/aggregator/target/rapids-4-spark-aggregator_2.12-21.12.0-SNAPSHOT-spark301db.jar not found.
```

also spark311db
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.7:deploy-file (default-cli) on project rapids-4-spark-parent: /home/jenkins/agent/workspace/rapids_databricks_nightly-dev-github-311-181/deploy/spark-rapids/aggregator/target/rapids-4-spark-aggregator_2.12-21.12.0-SNAPSHOT-spark311db.jar not found.
```

--- a/jenkins/databricks/deploy.sh
+++ b/jenkins/databricks/deploy.sh
@@ -30,7 +30,7 @@ VERSION_NUM=${BASE_SPARK_VERSION_TO_INSTALL_DATABRICKS_JARS//.}
 SPARK_VERSION_STR=spark$VERSION_NUM
 SPARK_PLUGIN_JAR_VERSION=`mvn help:evaluate -q -pl dist -Dexpression=project.version -DforceStdout`
 DB_SHIM_NAME=${SPARK_VERSION_STR}db
-DBJARFPATH=./aggregator/target/rapids-4-spark-aggregator_$SCALA_VERSION-$SPARK_PLUGIN_JAR_VERSION-${DB_SHIM_NAME}.jar
+DBJARFPATH=./aggregator/target/${DB_SHIM_NAME}/rapids-4-spark-aggregator_$SCALA_VERSION-$SPARK_PLUGIN_JAR_VERSION-${DB_SHIM_NAME}.jar
 echo "Databricks jar is: $DBJARFPATH"
 mvn -B deploy:deploy-file $MVN_URM_MIRROR -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \
     -Dfile=$DBJARFPATH -DpomFile=aggregator/dependency-reduced-pom.xml -Dclassifier=$DB_SHIM_NAME

